Assets/Scripts/Game/Controllers/LootController.cs

```diff
@@ -16,6 +19,37 @@ public enum LootType { Module, RareMetal}
 		/// </summary>
 		public int RareMetals { get; private set; }
 
+		/// <summary>
+		/// Data for creating dropped module loot
+		/// </summary>
+		[SerializeField]
+		public LootData moduleDropData;
+
+		/// <summary>
+		/// Data for creating dropped metal loot (appears on map, player can pick it up)
+		/// </summary>
+		[SerializeField]
+		public LootData rareMetalDropData;
+
+		/// <summary>
+		/// Max number of rare metals that will drop when an asteroid is destroyed
+		/// </summary>
+		[SerializeField, PropertyRange(1, 25)]
+		public int maxRareMetalDropAmount;
+
+		/// <summary>
+		/// Max number of modules that will drop when a ship is destroyed
+		/// </summary>
+		[SerializeField, PropertyRange(1, 25)]
+		public int maxModuleDropAmount;
+
+		PlayerShipEntity player; // here to test if giving parent will help w lootEntity destruction
```

Comment:
You can add `TODO: info here` before you comments and generally IDE's and sometimes compilers will highlight these and warn you if and where they are so that you can go back and chance them later. There's a TODO highlighter and linter for VSCode on the extensions store : )

Reply:
Ah very good point! Sorry I missed this >> I'll see if there's an equivalent plugin for Visual Studio